common/tools/dev-tool/src/commands/run/testNodeJSInput.ts

```diff
@@ -17,14 +17,21 @@ export const commandInfo = makeCommandInfo(
       default: false,
       description: "whether to run with test-proxy",
     },
+    "no-esm": {
+      shortName: "nesm",
+      kind: "boolean",
```

Comment:
Wasn't sure if this is the correct type, it is more of a switch than a boolean. If anyone knows a better way to support `--no-esm` translating to options["no-esm"] === true, _and_ options["no-esm"] === false if omitted I would be delighted to learn about it